### PR TITLE
Add procedure_cost to fulfillment model

### PR DESCRIPTION
**Pull request by BintLopez**
I rule and have completed some work on Case Manager that's ready for review!

This pull request makes the following changes:
* adds procedure_cost to Fulfillment model so that rake db:seed does not error out

It relates to the following issue #s: 
* Fixes https://github.com/DCAFEngineering/dcaf_case_management/issues/1540

### CODE REVIEW

**Comment by BintLopez** [comment]
Containers are failing to spin up at all it looks like?
image

**Comment by colinxfleming** [comment]
yeah we're having a problem with builds from forks. I'll add you to the github group in a bit.

**Comment by BintLopez** [comment]
cool coool! thanks @colinxfleming !

File: app/models/fulfillment.rb [hunk 1]
```diff
@@ -16,6 +16,7 @@ class Fulfillment
   field :check_number, type: String
   field :date_of_check, type: Date
   field :audited, type: Boolean
+  field :procedure_cost, type: Integer
```
Comment (colinxfleming) [inline app/models/fulfillment.rb]: this field is `fund_payout` now per #1529  -- `Fulfillment#procedure_cost` was conflicting with the field with the same name on `Patient`.

Let's change the field in `seeds` to be `fund_payout`, if that's cool?

sorry I missed this, thought I got it everywhere! my mistake.

**Comment by BintLopez** [comment]
Closing in favor of https://github.com/DCAFEngineering/dcaf_case_management/pull/1542

**Pull request closed without merging**